airbyte-protocol/models/src/main/java/io/airbyte/protocol/models/JsonSchemaPrimitive.java

```diff
@@ -4,11 +4,28 @@
 
 package io.airbyte.protocol.models;
 
+import com.google.common.collect.ImmutableMap;
+
 public enum JsonSchemaPrimitive {
-  STRING,
-  NUMBER,
-  OBJECT,
-  ARRAY,
-  BOOLEAN,
-  NULL;
+
+  STRING_DATE(ImmutableMap.of("type", "string", "format", "date")),
+  STRING_TIME(ImmutableMap.of("type", "string", "format", "time")),
+  STRING_TIMESTAMP(ImmutableMap.of("type", "string", "format", "date-time")),
```

Comment:
i recommend changing this to `STRING_DATETIME` as it matches the json primitive it represents

Reply:
> i recommend changing this to `STRING_DATETIME` as it matches the json primitive it represents

done